### ISSUE TITLE
Error opening file

### ISSUE THREAD

**Issue by maplelog** [bjorn/tiled#2755]
Hello! I've been unable to get tiled to work for a while. All it does when I try to open my .tbin file is to declare the following error:

![image](https://user-images.githubusercontent.com/61340546/75086430-6175ec80-5534-11ea-84bf-7d68752de52a.png)

I've tried getting it to work on several different computers, but the same thing happens. Could it have something to do with Windows? I've been able to use Tiled previously on a Windows system, but that's probably 2-3 years ago by now. I've tried both the newest release (1.3.2 at the time of writing) and the snapshot. I also have tbin.dll enabled in plugins.

Any help would be greatly appreciated.

**Comment by bjorn** [comment]
Does it only affect .tbin files?

I wonder if the error refers to the .tbin file you're trying to load, or whether it could be that it is trying to load a missing file. As-is, it sounds like some permissions issue. Can you open the .tbin file using any other editor?

It's unlikely that it would be causing issues, but could you try to copy the files to a path without special characters like "ú"?

**Comment by maplelog** [comment]
Yes, it seems to be affecting .tbin files only. I just noticed it popped up with a warning as well.

![image](https://user-images.githubusercontent.com/61340546/75595415-992ce900-5a8c-11ea-8aab-dfff5282003c.png)

Could this have something to do with it? I tried opening the same .tbin with tIDE as well, and I get the following error:

![image](https://user-images.githubusercontent.com/61340546/75595484-eb6e0a00-5a8c-11ea-9108-72c4a1fc5793.png)

To be fair, I don't know much about using tIDE, so it could just be a setting I haven't altered or something along those lines.
